src/components/SpiceItem.js

```diff
@@ -4,12 +4,13 @@ class SpiceItem extends React.Component {
 
   render() {
     return (
+      
       <div className="spice-item">
-        <img src={null /* replace me with an image url */} alt={"replace me with a TITLE"} />
+        <img src={this.props.image} alt={this.props.image} />
         <div className="details">
-          <h2>{"replace me with a TITLE"}</h2>
-          <p>{"replace me with a DESCRIPTION"}</p>
-          <em>{"replace me with NOTES"}</em>
+          <h2>{this.props.title}</h2>
+          <p>{this.props.description}</p>
+          <em>{this.props.notes}</em>
         </div>
       </div>
     )
```

Comment:
Great work! This would be a good place to practice using destructuring to make your return statement a little cleaner:


```jsx
render() {
    const { image, title, description, notes } = this.props
    return (
      <div className="spice-item">
        <img src={image} alt={title} />
        <div className="details">
          <h2>{title}</h2>
          <p>{description}</p>
          <em>{notes}</em>
        </div>
      </div>
    )
```